internal/importers/gvt/importer.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package gvt
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/golang/dep"
+	"github.com/golang/dep/internal/gps"
+	"github.com/golang/dep/internal/importers/base"
+	"github.com/pkg/errors"
+)
+
+const gvtPath = "vendor" + string(os.PathSeparator) + "manifest"
+
+// Importer imports gvt configuration into the dep configuration format.
+type Importer struct {
+	*base.Importer
+	gvtConfig gvtManifest
+}
+
+// NewImporter for gvt.
```

Comment:
So we don't forget, let's include in this comment that it also automatically handles gb as well.